packages/account/src/Components/trading-assessment/trading-assessment-existing-user.jsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Formik, Form } from 'formik';
+import {
+    FormSubmitButton,
+    Text,
+    Icon,
+    Button,
+    Modal,
+    DesktopWrapper,
+    MobileDialog,
+    MobileWrapper,
+} from '@deriv/components';
+import { generateValidationFunction, getDefaultFields, isMobile } from '@deriv/shared';
+import { localize, Localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import { trading_assessment_form_config } from 'Configs/trading-assessment-config';
+import TradingAssessmentRadioOption from './trading-assessment-radio-buttons';
+import TradingAssessmentDropdownOption from './trading-assessment-dropdown';
+
+const TradingAssessmentForm = ({ assessment_questions }) => {
+    const [current_question_index, setCurrentQuestionIndex] = React.useState(0);
+    const [is_next_button_enabled, setIsNextButtonEnabled] = React.useState(false);
+    const [current_question, setCurrentQuestion] = React.useState({});
+
+    const end_question_index = assessment_questions.length - 1;
+
+    React.useEffect(() => {
+        setCurrentQuestion(assessment_questions[current_question_index]);
+    }, []);
+
+    const handleNextButton = () => {
+        const next_question = current_question_index + 1;
+        if (next_question < assessment_questions.length) {
+            setCurrentQuestionIndex(next_question);
+            setCurrentQuestion(assessment_questions[next_question]);
+        }
+    };
+
+    const handlePrevButton = () => {
+        const prev_question = current_question_index - 1;
+        if (prev_question >= 0) {
+            setCurrentQuestionIndex(prev_question);
+            setCurrentQuestion(assessment_questions[prev_question]);
+        }
+    };
+
+    const handleValueSelection = (e, form_control, callBackFn, values) => {
+        if (typeof e.persist === 'function') e.persist();
+        callBackFn(form_control, e.target.value);
+        const latest_value = { ...values, [form_control]: e.target.value };
+        if (latest_value.risk_tolerance === 'No') {
+        }
+    };
+
+    const isAssessmentCompleted = answers => Object.values(answers).every(answer => Boolean(answer));
+
+    const hideElement = condition => {
```

Comment:
Suggestion: omit this function and use a statement directly in the styles. F.e. visibility: condition ? 'hidden' : 'visible'. I think the function is only used once in this file